spec.html

```diff
@@ -7610,6 +7591,20 @@
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-arguments-exotic-objects-hasproperty">
+        <h1>[[HasProperty]] ( _P_ )</h1>
+        <p>The [[HasProperty]] internal method of an arguments exotic object when called with property key _P_, performs the following steps:</p>
+        <emu-alg>
+          1. Let _args_ be the arguments object.
+          1. If _P_ is `"caller"`, then
+            1. Let _desc_ be OrdinaryGetOwnProperty(_args_, _P_).
+            1. If IsDataDescriptor(_desc_) is *true*, return *true*.
+          1. Return OrdinaryHasProperty(_args_, _P_).
+
+          NOTE: If an implementation does not provide a built-in caller property for argument exotic objects then step 2 of this algorithm must be skipped.
```

Comment:
Actually, it should just be a paragraph since it's normative. I should have looked at the other occurrence.